src/tv2-common/hotkeys/hotkey-defaults.ts

```diff
@@ -0,0 +1,226 @@
+import { SharedSourceLayers } from 'tv2-constants'
+import { literal } from '../util'
+import { GlobalHotkeyAssignments } from './global'
+import { RundownViewHotkeyAssignments } from './rundownView'
+import { ActiveSegmentHotketAssignments } from './segment'
+
+export interface TV2Hotkeys {
+	activeSegment: ActiveSegmentHotketAssignments
+	global: GlobalHotkeyAssignments
+	rundownView: RundownViewHotkeyAssignments
+}
+
+export const defaultHotkeys: TV2Hotkeys = {
+	activeSegment: literal<ActiveSegmentHotketAssignments>({
+		lowerThirds: ['KeyA', 'KeyS', 'KeyD', 'KeyF', 'KeyG']
+	}),
+	global: literal<GlobalHotkeyAssignments>({
+		recallLast: {
+			DVE: 'F10',
+			Live: 'Ctrl+Alt+Shift+KeyB'
+		},
+		takeWithTransition: ['NumpadDivide', 'NumpadSubtract', 'NumpadAdd'],
+		camera: {
+			directCut: ['F1', 'F2', 'F3', 'F4', 'F5'],
+			queue: ['Ctrl+Shift+F1', 'Ctrl+Shift+F2', 'Ctrl+Shift+F3', 'Ctrl+Shift+F4', 'Ctrl+Shift+F5'],
```

Comment:
It should be `Control` instead of `Ctrl` https://github.com/nrkno/sorensen/blob/main/README.md